### ISSUE TITLE
1.8 - Unable to join Observers using "/join o"

### ISSUE THREAD

**Issue by SuicuneScale** [OvercastNetwork/Issues#1819]
Title. When performed on a 1.8 server (CTF), it gives the error message "no teams match query". The issue is present even when there is no orange team on the map playing. This still works on the 1.7 server, however.

Came from this:

https://oc.tc/forums/topics/55413715c29635691f0015ba

**Comment by samtheman43** [comment]
just for easier clarafication, I'd add [CTF] to the title.

**Comment by Shadowpat-15** [comment]
Thank you for posting this for me, Suicune. Actually, /join o is what isn't working, not /join obs. /join obs seems to work fine, but even when there is not orange team, /join o will not direct you to the observing team like it will on other servers. See the forum post for details :)

**Comment by SuicuneScale** [comment]
Sorry, I read it wrong. Let me rephrase that, then.

**Comment by kroest** [comment]
i want to add on this, when you enter /join obs, it doenst tell you that you joined the observers.

**Comment by CaptainElliott** [comment]
Is this on all maps?

**Comment by Pablete1234** [comment]
@CaptainElliott yes, it's basically 1.8 plugin thing, before just /join o ment /join observers, now you need to do /join obs

**Comment by Electroid** [comment]
The command was changed to `/leave`. `/join obs` exists for legacy support only.
